drivers/spi/spi_mcux_lpspi.c

```diff
@@ -485,6 +508,13 @@ static int transceive(const struct device *dev,
 	struct spi_mcux_data *data = dev->data;
 	int ret;
 
+	ret = spi_mcux_configure(dev, spi_cfg);
+	if (ret != 0) {
+		goto out;
+	}
+
+	spi_context_cs_control(&data->ctx, true);
+	spi_context_cs_control(&data->ctx, false);
```

Comment:
Why these switch on/off of the CS line? 

and why the spi_mcux_configure() call, since it's already called below at line 520 ?

Fidally, how these changes fit with RTIO addition in this driver?

Reply:
Thank you, I don't know where this came from, I have removed this.

Reply:
FYI you just pushed and it's not removed dont know if it's intentional

Reply:
Thank you again, it is now removed.